db/db_impl_readonly.cc

```diff
@@ -52,11 +52,17 @@ Status DBImplReadOnly::Get(const ReadOptions& read_options,
                               &max_covering_tombstone_seq, read_options)) {
     pinnable_val->PinSelf();
     RecordTick(stats_, MEMTABLE_HIT);
+    PERF_COUNTER_BY_LEVEL_ADD(user_key_return_count, 1, 0);
   } else {
     PERF_TIMER_GUARD(get_from_output_files_time);
+    int level_found = -1;
     super_version->current->Get(read_options, lkey, pinnable_val, &s,
-                                &merge_context, &max_covering_tombstone_seq);
+                                &merge_context, &level_found,
```

Comment:
Version::Get() to return level found makes the function less clean. Do we really need to return it, or can be update the counter inside Version::Get()?

Reply:
Sure, we can update the counter inside `Version::Get()`, doesn't have to happen in `DBImpl::GetImpl`